Dockerfiles/Dockerfile_Ubuntu

```diff
@@ -0,0 +1,20 @@
+FROM ubuntu:23.10 
+
+RUN apt-get update
+RUN apt-get install -y build-essential
+RUN apt-get install -y netcdf-bin libnetcdf-c++4-dev libboost-all-dev libeigen3-dev cmake
+RUN apt-get install -y python3-netcdf4
+
+WORKDIR /home/nextsim
+COPY . /home/nextsim
+
+RUN rm -r build
+RUN mkdir -p /home/nextsim/build 
+WORKDIR /home/nextsim/build 
+RUN cmake -B/home/nextsim/build  -H/home/nextsim/ >&2
```

Comment:
why do you redirect output to `stderr` i.e., `>&2`?

Reply:
Because I like to see what happens, mainly for debugging purposes/to make sure everything works as expected. I can delete that if you think it is not good style.

Reply:
ok, I removed that in both files